Time/WorldTimeGrid.10s.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
```

Comment:
One suggestion... Might consider using this for the shebang:

```suggestion
#!/usr/bin/env -S PATH="${PATH}:/opt/homebrew/bin:/usr/local/bin" python3
```

Allows the plugin to pick up python3 from either macOS install, Homebrew on M1, or Homebrew on Intel.